src/Microsoft.Identity.Client/AppConfig/IAppConfig.cs

```diff
@@ -81,6 +81,10 @@ public interface IAppConfig
         /// </summary>
         string Claims { get; }
 
+        /// <summary>
+        /// </summary>
+        bool IsBrokerEnabled { get; }
```

Comment:
Hide it like ClientCredential

Reply:
i suppose by like you mean similar to? I would expect this one to be relevant for shared, ios and android ?

---
In reply to: [254633519](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/874#discussion_r254633519) [](ancestors = 254633519)